### DISCUSSION TITLE
where are my prizes??

### DISCUSSION THREAD

**Discussion by alisseo** [streetcomplete/StreetComplete#2983] [Q&A] [answered]
I am a happy user with a lot of completed missions; now I am "Inspector" with 8 "levels" already done.
for each level i received a good link as a reward but now i can't see this link. now i need this link, which i only tried when i won.
Thank you

**Comment by westnordost** [comment, marked as answer]
Press the Hamburger menu in the upper right, then tap on Profile. In this screen, the unlocked links are in the rightmost tab.

If you haven't seen the Profile screen yet, there is more to discover there, have fun! ❤️

**Reply by alisseo** [reply]
yes i have the prize  list, but the links winned is invisible.
Sorry for the delay. a nice time!:)

**Reply by westnordost** [reply]
What android version are you using? Are animations off or on? Do you have some achievements unlocked alrea?dy

With invisible, you mean the links list is empty?

**Reply by alisseo** [reply]
i'm in android 9 and my animations is off. i have many achievements unlocked, i have only prize list but not the links winned

**Reply by westnordost** [reply]
so the tab, what does it say? Does it say that the list is empty?

**Reply by westnordost** [reply]
what if you turn on animations, will the links be shown then?
